types/office-js-preview/index.d.ts

```diff
@@ -26752,38 +26743,37 @@ declare namespace Excel {
         context: RequestContext;
         /**
          *
-         * HTML color code representing the color of the background, of the form #RRGGBB (e.g. "FFA500") or as a named HTML color (e.g. "orange")
+         * HTML color code representing the color of the background, of the form #RRGGBB (e.g., "FFA500") or as a named HTML color (e.g., "orange")
          *
          * [Api set: ExcelApi 1.1]
          */
         color: string;
         /**
          *
-         * Gets or sets the pattern of a Range. See Excel.FillPattern for details. LinearGradient and RectangularGradient are not supported.
+         * The pattern of a range. See Excel.FillPattern for details. LinearGradient and RectangularGradient are not supported.
                     A null value indicates that the entire range doesn't have uniform pattern setting.
          *
          * [Api set: ExcelApi 1.9]
          */
         pattern: Excel.FillPattern | "None" | "Solid" | "Gray50" | "Gray75" | "Gray25" | "Horizontal" | "Vertical" | "Down" | "Up" | "Checker" | "SemiGray75" | "LightHorizontal" | "LightVertical" | "LightDown" | "LightUp" | "Grid" | "CrissCross" | "Gray16" | "Gray8" | "LinearGradient" | "RectangularGradient";
         /**
          *
-         * Sets HTML color code representing the color of the Range pattern, of the form #RRGGBB (e.g. "FFA500") or as a named HTML color (e.g. "orange").
-                    Gets HTML color code representing the color of the Range pattern, of the form #RRGGBB (e.g. "FFA500").
+         * The HTML color code representing the color of the range pattern, of the form #RRGGBB (e.g., "FFA500").
```

Comment:
Does named HTML color no longer apply for this and other places?

Reply:
It does. The change was made to remove the get/set difference and the named example was removed. I'll add it back in here.